src/main/java/br/com/digituz/mailer/model/Emails.java

```diff
@@ -0,0 +1,73 @@
+package br.com.digituz.mailer.model;
+
+import java.util.List;
+
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * @author daniel
+ */
+@Entity
```

Comment:
Erraaaaado errado não está. Mas tem formas mais seguras de lidar com o esquema de banco de dados. O hibernate com certeza facilita muito, mas eu prefiro ter um controle mais próximo sobre como as tabelas, indíces e afins são gerados.

Dá uma lida [nesse artigo onde eu falo sobre Flyway](https://github.com/brunokrebs/blog/blob/00868614549770c0475b9b4070adf31bcfdcde66/_posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown).